pandas/tests/plotting/test_hist_method.py

```diff
@@ -34,14 +34,14 @@ def test_hist_legacy(self):
             _check_plot_works(self.ts.hist, by=self.ts.index.month, bins=5)
 
         fig, ax = self.plt.subplots(1, 1)
-        _check_plot_works(self.ts.hist, ax=ax)
-        _check_plot_works(self.ts.hist, ax=ax, figure=fig)
-        _check_plot_works(self.ts.hist, figure=fig)
+        _check_plot_works(self.ts.hist, ax=ax, default_axes=True)
+        _check_plot_works(self.ts.hist, ax=ax, figure=fig, default_axes=True)
+        _check_plot_works(self.ts.hist, figure=fig, default_axes=True)
         tm.close()
 
         fig, (ax1, ax2) = self.plt.subplots(1, 2)
-        _check_plot_works(self.ts.hist, figure=fig, ax=ax1)
-        _check_plot_works(self.ts.hist, figure=fig, ax=ax2)
+        _check_plot_works(self.ts.hist, figure=fig, ax=ax1, default_axes=True)
+        _check_plot_works(self.ts.hist, figure=fig, ax=ax2, default_axes=True)
 
```

Comment:
`default_axes=False` was causing plots on additional subplots in addition to the passed `ax`, redundant with all the existing figure + axis plotting calls already in this method. `default_axes=True` just makes sure only the passed `ax` is used.